### PR TITLE
Added claude 3.5 sonnet and perplexity with llama model to curated ai…

### PR DESCRIPTION
**Pull request by krishkalaria12**
Added claude 3.5 sonnet model and perplexity with llama large 8b instruct model to the curated list of models

### CODE REVIEW

File: models/claude-3-5-sonnet.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Anthropic } from '@anthropic-ai/sdk';
+import { type ModelHandler } from ".";
+
+const anthropic = new Anthropic({
+    apiKey: process.env.ANTHROPIC_API_KEY,
+});
```
[the remark this answers is not in the document]
Comment (delasy) [reply models/claude-3-5-sonnet.ts:6]: having schemas in main thread is fine though

File: models/claude-3-5-sonnet.ts
Comment (delasy) [inline models/claude-3-5-sonnet.ts:1]: in this file code style is different from entire app

**Comment by krishkalaria12** [comment]
I have updated the file structure in the PR with the handler moved and also removing the try catch from the sonnet file making it not suppress the error

**Comment by FleetAdmiralJakob** [comment]
You added axios without using it somewhere. Is that a mistake or did you do it on purpose and if the latter why?

**Comment by krishkalaria12** [comment]
I added axios because perplexity doesn't offer any sdk or package for using the models and it was only available through the api call and i think axios is the way to go for calling api's rather the fetch method where you need to do the heavy lifting work and axois provides things out of box which normal way of calling api's using fetch doesn't

**Comment by krishkalaria12** [comment]
If axios doesn't work out for us. i can do the normal standard way using the fetch too

**Comment by krishkalaria12** [comment]
Sure will change it to fetch from axios

**Comment by webdevcody** [comment]
@krishkalaria12 btw are you in discord?

**Comment by krishkalaria12** [comment]
@webdevcody my discord username is krishkalaria12

**Pull request merged**